mmv1/third_party/terraform/services/gemini/iam_gemini_repository_group_test.go.tmpl

```diff
@@ -0,0 +1,290 @@
+package gemini_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+// To run tests locally please replace the `oauth_token_secret_version` with your secret manager version.
+// More details: https://cloud.google.com/developer-connect/docs/connect-github-repo#before_you_begin
+
+func TestAccGeminiRepositoryGroupIamBinding(t *testing.T) {
```

Comment:
Were you unable to get the IAM tests to generate? We shouldn't need handwritten test files like this if the IAM generation works as expected

Reply:
Replaced handwritten tests with generated ones

Reply:
It seems that IAM tests that generate with `t.Parallel` still cause conflicts on the API side, so until turning it off in generating tests is possible, we will go with handwritten ones, which are the same but not parallel.